main.js

```diff
@@ -0,0 +1,254 @@
+const playersArray = [];
```

Comment:
**REQUIRED.**
* It doesn't use global code - encapsulate logic inside modules or factories ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/global_requirement.png) from Odin)